macros/Trouble Shooting/default.md

```diff
@@ -419,6 +419,32 @@ Premigration checks are done before initiating any migration. For warm or TBC mi
 
 If a cdrom drive is not present we will display a warning that a cdrom drive will be added. However if the “prepare-to-migrate” has not been run this warning overrides this check and the migration can proceed. However since the prepare-to-migrate has not been run the migration will eventually fail. (Ref:CP-5713)
 
+ifdef(~sTARGET_VCD~e, ~s
+
+###### **Static warm migration fails for VMs with mutiple NICs on filling same static ip on all the NICs from GUI **
```

Comment:
Title need to update something like,

Migration with static IP allocation will fail if VM has Multiple network adaptors (min 2) connected to the same network.

Reply:
Updated the title, please check now.